Directory.Build.props

```diff
@@ -9,6 +9,9 @@
 
     <!-- Don't make missing XML docs a fatal build error, but still surface so we have visibility into undocumented APIs. -->
     <WarningsNotAsErrors>$(WarningsNotAsErrors);CS1591</WarningsNotAsErrors>
+
+    <!-- static local methods are a C# 8.0 feature -->
+    <LangVersion>preview</LangVersion>
```

Comment:
Should we use 8.0 instead of preview to be more explicit?

Reply:
8.0 creates warnings about static local methods. Will re-evaluate closer to 3.0